buptelecmon/main.py

```diff
@@ -48,6 +53,16 @@ def output(dormitory, data, params=None):
             if convert_to_float(data['pTotal']) != 0 else 'Infinite')
     )
 
+def output_shahe(data, params=None):
+    # Shahe Campus's data format is different from Xitucheng Campus
+    print('%s %s - Surplus money: %.2f Yuan\t Total power consumption: %.2f kWh).' %
```

Comment:
Redunant parenthese `)` in `%.2f kWh)`, please remove it 😄

Reply:
Sorry, didn't note this.